requirements-dev.txt

```diff
@@ -8,4 +8,4 @@ aiohttp
 requests
 cbor2
 behave
--e git+https://github.com/pubnub/vcrpy.git@aiotthp_redirect_enabled#egg=vcrpy
+vcrpy
```

Comment:
from the intel I managed to gather our fork of `vcrpy` was made to support Python's v2.7 branch. Current Python runs well with the "factory" version of this library